Assets/Scripts/MVC/TankService.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Outscal.BattleTank
+{
+    /// <summary>
+    /// service class that 
+    /// </summary>
+    public class TankService : MonoBehaviour
```

Comment:
Must be a singleton class  that should be inherited from generic singleton class